### ISSUE TITLE
Dropdown keyboard navigation does not work if I use getter 

### ISSUE THREAD

**Issue by darkcris1** [ng-bootstrap/ng-bootstrap#4276]
[Reproduction](https://stackblitz.com/edit/angular-qdfsqu?file=src/app/dropdown-basic.ts)

Angular: 13.x

ng-bootstrap: 12.x

Bootstrap: 5

**Comment by jnizet** [comment]
Hi @darkcris1 

By using a getter like that, every time Angular triggers a change detection, you create a new array containing new options. You shouldn't do that.
